packages/asc-ui/src/components/TopBar/StyledTopBar.ts

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components'
+
+const defaultBackgroundColor = '#fff'
+
+export type Props = {
+  backgroundColor?: string
+}
+
+const StyledTopBar = styled.header<Props>`
+  display: flex;
+  position: relative;
+  align-items: center;
+  z-index: 0;
+  min-height: 54px;
+  padding: 0 15px;
+  background-color: ${props =>
```

Comment:
The props are destructured here in most of the codebase

Reply:
done